idunn/places/poi.py

```diff
@@ -82,7 +78,49 @@ def get_contribute_url(self):
             return None
 
     def get_source(self):
-        return self.source
+        pass
+
+
+class OsmPOI(POI):
+    @cached_property
```

Comment:
I'd argue that a `.rsplit` probably doesn't cost enough to justify using `@cached_property` (which could lead to surprising behaviors), but it wasn't introduced here anyway :3